R/mobr.R

```diff
@@ -427,7 +427,305 @@ samp_ssad = function(comm, groups){
   return(comm_out)
 }
 
+# Convert specified columns of a dataframe from factors to numeric
+df_factor_to_numeric = function(dataframe, cols = NULL){
+    if (is.null(cols)) cols = 1:ncol(dataframe)
+    for (col in cols){
+        if ('factor' %in% class(dataframe[, col]))
+            dataframe[, col] = as.numeric(levels(dataframe[, col]))[dataframe[, col]]
+    }
+    return(dataframe)
+}
+
```

Comment:
This function only converts factors to numeric; it doesn't deal with characters. 
For your data frame you'd want `df_factor_to_numeric(tst, 2)`.